doc/userguide/security.rst

```diff
@@ -0,0 +1,137 @@
+Security Considerations
+=======================
+
```

Comment:
should we have a quick intro explaining *why* it matters to do so?

Reply:
I think I can into that under running as root, but will add a small blurb.

Reply:
For now...

```
Suricata is a security tool that processes untrusted network data, as
well as requiring elevated system privileges to acquire that
data. This combination deserves extra security pre-cautions that we
discuss below.

Additionally, supply chain attacks, particularly around rule
distribution could potentially target Suricata installations.
```